src/globals/scss/_typography.scss

```diff
@@ -1,7 +1,7 @@
 @import 'vars';
 
 $font-family-mono: 'ibm-plex-mono', 'Menlo', 'DejaVu Sans Mono', 'Bitstream Vera Sans Mono', Courier, monospace !default;
-$font-family-sans-serif: 'ibm-plex-sans', Helvetica Neue, Arial, sans-serif !default;
+$font-family-sans-serif: 'IBM Plex Sans', 'ibm-plex-sans', Helvetica Neue, Arial, sans-serif !default;
```

Comment:
Wanted to double-check if this is a temporary change, or we really had an issue with fonts.

Reply:
Italic fonts were not rendering for me unless I added this in 🤷‍♂️